### PR TITLE
UI pause

### PR DESCRIPTION
**Pull request by PaTo81013**
(https://youtu.be/qEyNLcZ7ras)

### CODE REVIEW

File: Where_Is_My_Wife/Assets/_Main/Scripts/UI/PauseController.cs [hunk 1]
```diff
@@ -37,14 +37,14 @@ private void PauseGame()
...
-            Time.timeScale = 0;
+            //Time.timeScale = 0;
         }
         
         private void ResumeGame()
         {
             paused = false;
             _canvas.SetActive(false);
-            Time.timeScale = 1;
+            //Time.timeScale = 1;
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/UI/PauseController.cs:47]: Why? the code to remove this is on @FerchusGames branch

Comment (Horothenic) [reply Where_Is_My_Wife/Assets/_Main/Scripts/UI/PauseController.cs:47]: Nvm the video explained why

**Comment by Horothenic** [comment]
The video never shows how you click Pause button and the menu appears

**Comment by PaTo81013** [comment]
> The video never shows how you click Pause button and the menu appears

**Review by Horothenic — APPROVED** [review]

**Pull request merged**